docker/python3-ubi/Dockerfile

```diff
@@ -98,6 +98,8 @@ Options = UnsafeLegacyRenegotiation\n' > /tmp/ssl.cnf \
 RUN groupadd -g 4000 demisto && \
     useradd -u 4000 -g demisto demisto -s /bin/sh
 
+ENV PYTHON_VERSION=$(python3 -c 'import platform; print(platform.python_version())')
```

Comment:
does this work? thought it cant be dynamic.
I think we should do ARG PYTHON_VERDION=3.10.8 that we use for this variable and also for the download and install iteslf. Or maybe use ENV if the env cant depend on the arg?

Reply:
you are right it can't be dynamic